includes/class-newspack-popups-inserter.php

```diff
@@ -33,22 +57,23 @@ public static function popup( $content = '' ) {
 		if ( is_user_logged_in() || ! is_single() ) {
 			return $content;
 		}
-		/* End */
```

Comment:
On line 45 you have `add_action( 'the_content', [ $this, 'popup' ] );` It should be `add_filter` instead.